packages/ui-extensions-dev-console/src/App.tsx

```diff
@@ -42,6 +45,22 @@ function App() {
           <AppProvider i18n={enTranslations}>
             <Layout>
               <Routes />
+              <ToastContainer
```

Comment:
Minor, but have you considered extracting this in a component and adding it below the `<main>{children}</main>` line in the `Layout` component? Tipically `App.tsx` is quite lean and just a collection of providers/components.